pyclesperanto_prototype/_tier9/_statistics_of_image.py

```diff
@@ -2,7 +2,7 @@
 from .._tier0 import plugin_function
 
 @plugin_function
-def statistics_of_image(image : Image):
+def statistics_of_image(image : Image, use_gpu:bool = False):
```

Comment:
Note, for arguments like this one, I would recommend `(image : Image, *, use_gpu : bool = False)`, which forces users to use keyword arguments, `statistics_of_image(my_image, use_gpu=True)`. This has two advantages:

- user code is more self-documenting. Compare the above to `statistics_of_image(my_image, True)` — what does `True` mean here???
- if you ever want to deprecate the use of that argument, it becomes much easier to track whether user code intended to pass that argument or some future positional argument, e.g. `statistics_of_image(image : Image, mask : Image = None, *, use_gpu : bool = False)` becomes easier to get to without breaking user code.

Reply:
> which forces users to use keyword arguments

keyword arguments are not supported in the other clEsperanto dialects 😉 At least the mandatory arguments must be accessible via the minimal `operation(input1, input2)` syntax. I see what you mean. Maybe there is a way for improving tha API here..

Reply:
> must be accessible

I guess that's a problem for code generation, since exact uniformity is not possible? (?) But it should be possible to do something different for generated Python code vs generated [insert language I don't care about] code? 😜